airflow/providers/google/cloud/sensors/cloud_composer.py

```diff
@@ -59,6 +64,13 @@ def __init__(
         pooling_period_seconds: int = 30,
         **kwargs,
     ):
+        warnings.warn(
+            f"The `{self.__class__.__name__}` operator is deprecated. You can achieve the same functionality "
+            f"by using operators in deferrable or non-deferrable mode, since every operator for Cloud "
```

Comment:
Which operator? lets be specific

Reply:
Hi :)
So literally every operator in Cloud Composer (create/update/delete). According to the API and my tests, there is no need to call sensor after executing those operators from Cloud Composer, because the waiting operation is done in hooks. When trying to retrieve operation_id after executing operators, the operation has already ended and that leads to error.
I will update the message in deprecate warning to be more specific